includes/class-widget.php

```diff
@@ -93,13 +93,13 @@ public function widget( $args, $instance ) {
 
 			if ( $is_amp ) {
 				?>
-					<amp-lightbox id="republication-tracker-tool-modal" layout="nodisplay" role="dialog" aria-modal="true">
+					<amp-lightbox id="republication-tracker-tool-modal" layout="nodisplay" role="dialog" aria-modal="true" aria-labelledby="republish-modal-label">
```

Comment:
Nit: This probably shouldn't be an `amp-lightbox` anymore, right? Should it be a regular `div` now that we're non-AMP-compliant?

Reply:
There's [actually both in the code](https://github.com/Automattic/republication-tracker-tool/blob/master/includes/class-widget.php#L94-L105) (switched out using the `is_amp` check), and I added the attribute to both 😬 Assuming there's not a large audience using this plugin that still need AMP support, it's probably safe to remove that plus the couple other related lines [like this](https://github.com/Automattic/republication-tracker-tool/blob/721d54fdb09549d74a8294a1253d9d378414667a/includes/shareable-content.php#L102) in a future commit!